src/consumer/component/sources/dist.js

```diff
@@ -51,7 +50,7 @@ export default class Dist extends Vinyl {
 
   buildSrcWithSourceMapAnnotation(fileName: string) {
     return this.mappings ?
-    `${this.contents.toString()}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.contents.toString();
+    `${this.compiledContent.toString()}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.contents.toString();
```

Comment:
the logic checks if the build returned mappings and if so it writes to compiled contents to the dist.
otherwise it write the current content to the dist file